libs/template/materialize.go

```diff
@@ -72,6 +66,16 @@ func Materialize(ctx context.Context, configFilePath, templateRoot, outputDir st
 		return err
 	}
 
+	// Print welcome message
+	welcome := config.schema.WelcomeMessage
+	if welcome != "" {
+		welcome, err = r.executeTemplate(welcome)
+		if err != nil {
+			return err
+		}
+		cmdio.LogString(ctx, welcome)
+	}
```

Comment:
Note that this PR doesn't make use of this change yet since that would mean it would only work in newer CLI versions. Since older CLI versions don't support `{{workspace_host}}` in the welcome message, the templates put it in the first question instead.

Reply:
You could use it here since it's a built-in template.

External templates will need to specify the minimum CLI version to use.

Reply:
Ah, yes, that's right. So the only downside would be if people copy-paste it, then it might not work with older versions. I think we can live with that.

I'll change the template to just make use of it in the welcome text.